common/pom.xml

```diff
@@ -37,6 +67,10 @@
       <scope>compile</scope> <!-- override the 'test' scope defined at parent pom.xml -->
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>org.jctools</groupId>
+      <artifactId>jctools-core</artifactId>
+    </dependency>
```

Comment:
Should be done automatically by the shade plugin, it creates a dependency reduced pom.